### PR TITLE
Related chart defaults to use percentage

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

**Comment by beets** [comment]
can we fix this in the config instead? we have some charts where this won't make sense, eg
government debt per capita
retail drug distribution per capita

i think there are only a handful that doesn't have this set where it applies, all in economics:
state unemployment insurance rate
percentage of people employed
labor force participation rate

**Comment by shifucun** [comment]
Fixed the chart config now.

File: server/chart_config.json [hunk 1]
```diff
@@ -9,15 +9,17 @@
   },
   {
     "category": "Economics",
-    "title": "State Unemployment Insurance",
+    "title": "State Unemployment Insurance Claim",
```
Comment (beets) [inline server/chart_config.json:12]: nit (here and related title): Claims

File: server/chart_config.json [hunk 2]
```diff
@@ -894,7 +907,7 @@
     ],
     "isOverview": true,
     "relatedChart": {
-      "title": "Marital Status Per Capita",
+      "title": "Marital Status Rate",
```
Comment (beets) [inline server/chart_config.json:910]: citizenship and other pop breakdowns are currently listed as "per capita" so this was meant to be consistent with those.

**Review by beets — COMMENTED** [review]
thanks for the udpate!

File: server/chart_config.json (hunk 2 above)
Comment (shifucun) [reply server/chart_config.json:910]: Find it a bit confusing if we should 20% together with "per capita", maybe choose one of the two, I feel rate is closer to what we use daily?

Comment (beets) [reply server/chart_config.json:910]: i figure for demographics related stats, per capita is more accurate..

Comment (beets) [reply server/chart_config.json:910]: let's keep rates:
http://www.oecd.org/social/family/SF_3_1_Marriage_and_divorce_rates.pdf

**Review by beets — APPROVED** [review]
could you update claim to claims? then this is good to go

File: server/chart_config.json (hunk 2 above)
Comment (shifucun) [reply server/chart_config.json:910]: Done. Thanks!

**Pull request merged**